types/canonical_json.go

```diff
@@ -22,13 +27,15 @@ type CanonicalJSONProposal struct {
 	POLBlockID       CanonicalJSONBlockID       `json:"pol_block_id"`
 	POLRound         int                        `json:"pol_round"`
 	Round            int                        `json:"round"`
+	Timestamp        string                     `json:"timestamp"`
```

Comment:
Maybe better to use an int here ?

Reply:
Hmmm....

Reply:
Int would not work, it must be int64 to handle milliseconds. Then we define it as what? Milliseconds since unix epoch?

I feel the json encoding is designed to be somewhat human readable. The random int in the rpc interface for header is also quite confusing.